packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -114,7 +114,10 @@ const AppNotificationMessages = ({
                   'identity',
               ].includes(message.key) || message.type === 'p2p_completed_order'
             : true;
-        return is_not_marked_notification && is_non_hidden_notification;
+
+        const is_only_for_p2p_notification =
+            window.location.pathname !== routes.cashier_p2p || message?.platform === 'P2P';
+        return is_not_marked_notification && is_non_hidden_notification && is_only_for_p2p_notification;
```

Comment:
Here is where we ensure that only P2P related notification cards are shown within P2P, this is to avoid notification cards such as `2_f_a` and `poa_poi_verified` to be shown in P2P, except ONLY in the notification bell (when user clicks the notification bell it should display non P2P related notification as well)